AppBuilder/platform/views/viewComponent/ABViewFormDatepickerComponent.js

```diff
@@ -71,7 +71,7 @@ module.exports = class ABViewFormDatepickerComponent extends (
          _ui.value = new Date(_ui.value);
 
       // if we have webix locale set, will use the date format form there.
-      if (field !== null && !window.webixLocale) _ui.format = field.getFormat();
+      if (!window.webixLocale) _ui.format = field.getFormat();
```

Comment:
Again, same issue with `field` being `undefined` or `null`.  without that check, `field.getFormat()` will crash.

Reply:
We already return `if (!field)` on line 57